src/example.gpl

```diff
@@ -1,3 +1,5 @@
 fn empty() {}
 
 fn take_args(a int, b int) {}
+
+fn main() { empty() }
```

Comment:
Maybe we should add this file to the .gitignore so we can freely test code

Reply:
Yeah, this would make things much easier. 
Since we now have some documentation and a roadmap, this file is not needed for the original purposes.

Reply:
I've added a filter for all gpl files in src. This will help when we need to test multi-file features

Reply:
I've added a filter for all gpl files in src. This will help when we need to test multi-file features